src/test/java/com/github/thorbenkuck/netcom2/pipeline/QueuedReceivePipelineTest.java

```diff
@@ -355,6 +355,7 @@ public void ifClosed1() throws Exception {
 		pipeline.ifClosed(() -> {
 			throw new PipelineAccessException("");
 		});
+		assertTrue(!pipeline.isClosed());
```

Comment:
Is this really necessary?
Why not
```
assertFalse(pipeline.isClosed());
```
?
In my opinion, it reads more naturally.